src/platform/windows/window.rs

```diff
@@ -715,15 +715,37 @@ unsafe fn init(
         events_loop_proxy,
     };
 
-    if let Some(_) = attributes.fullscreen {
-        win.set_fullscreen(attributes.fullscreen);
+    if let Some(_) = &attributes.fullscreen {
+        win.set_fullscreen(attributes.fullscreen.clone());
         force_window_active(win.window.0);
     }
 
     if let Some(dimensions) = attributes.dimensions {
         win.set_inner_size(dimensions);
     }
 
+    if attributes.center_window.is_some() && !&attributes.fullscreen.is_some(){
```

Comment:
We can replace all this code with something along the lines of this, right (pseudo code):
```
if attributes.fullscreen.is_none() {
    if let Some(mon) = attributes.center_window {
        win.set_centered(mon);
    }
}
```